python-pytest.el

```diff
@@ -116,11 +116,11 @@ When non-nil only ‘test_foo()’ will match, and nothing else."
 (defvar-local python-pytest--current-command nil
   "Current command; used in python-pytest-mode buffers.")
 
-(fmakunbound 'python-pytest-popup)
-(makunbound 'python-pytest-popup)
+(fmakunbound 'python-pytest-dispatch)
+(makunbound 'python-pytest-dispatch)
```

Comment:
this was a hack required for `magit-popup` to be able to make `eval-buffer` actually overwrite the previous definitions. is it still needed with `transient.el`?

Reply:
I'm not sure. When I developed the change I simply evaluated `define-transient-command` below and that was replacing definition. I can check if that works with `eval-buffer`